imports/ui/unit-explorer/unit-explorer.jsx

```diff
@@ -43,13 +49,14 @@ class UnitExplorer extends Component {
                 ))}
               </div>
             </div>
-            <Link to='/unit/new' className='link'>
-              <MenuItem className='no-shrink' innerDivStyle={{padding: 0}}>
-                <div className='tc bondi-blue fw5 bt b--moon-gray'>
-                  Add Unit
-                </div>
-              </MenuItem>
-            </Link>
+            <div className='absolute bottom-2 right-2'>
+              <FloatingActionButton
```

Comment:
You should use the values from `fabDescriptor` directly here, and remove it.
The added benefit of that is that when you look at this code later, you won't need to scroll up and look for `fabDescriptor`. You'll know what this button does just by looking at these lines.